spp_change_request/models/change_request.py

```diff
@@ -152,6 +152,24 @@ def _default_name(self):
         compute="_compute_validation_group_id",
         store=True,
     )
+    hide_button_css = fields.Html(
+        string="CSS", sanitize=False, compute="_compute_hide_button_css", store=False
+    )
```

Comment:
Try the javascript solution in my comment.

Reply:
As @nhatnm0612 suggested, you can try adding a record rule that is applied only to 'edit' for addressing the requirement of allowing the editing of fields only for users assigned to the CR.